ui/DesignSystem/Component/AdditionalActionsDropdown.svelte

```diff
@@ -18,7 +18,7 @@
     disabled?: boolean;
   }
   export let menuItems: MenuItem[];
-  export let headerTitle: string;
+  export let headerTitle: string = "";
```

Comment:
I think this is a bit more idiomatic:
```diff
diff --git a/ui/DesignSystem/Component/AdditionalActionsDropdown.svelte b/ui/DesignSystem/Component/AdditionalActionsDropdown.svelte
index ec32f7c8..3faeb18a 100644
--- a/ui/DesignSystem/Component/AdditionalActionsDropdown.svelte
+++ b/ui/DesignSystem/Component/AdditionalActionsDropdown.svelte
@@ -18,7 +18,7 @@
     disabled?: boolean;
   }
   export let menuItems: MenuItem[];
-  export let headerTitle: string = "";
+  export let headerTitle: string | undefined = undefined;

   export let dataCy = "";
   export let style = "";
@@ -133,7 +133,7 @@
     </button>
     {#if expanded}
       <div out:fade={{ duration: 100 }} class="modal" hidden={!expanded}>
-        {#if headerTitle !== ""}
+        {#if headerTitle}
           <div class="header">
             <StyledCopyable value={headerTitle} />
           </div>
```

Reply:
Weird magic, but I like it.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1940/commits/00574b15ac98df5ad530393b890bc2d68b8bd11a